.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Comment:
Weekly seems a little too frequent here. We don't cut releases weekly, so no one will really be using weekly updates and I don't know that I want to review weekly dependency updates. Monthly would probably be sufficient as it is a little closer to our release cadence.

Reply:
Changed it to monthly.